packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/dashboard/translations/Messages_fr_FR.json

```diff
@@ -36,5 +36,6 @@
   "telephony_group_billing_dashboard_RMA_status_received": "Réception du colis",
   "telephony_group_billing_dashboard_no_phone": "Ligne nue",
   "telephony_group_billing_dashboard_RMA_title": "RMA",
-  "telephony_group_billing_dashboard_calls": "Appels"
-}
\ No newline at end of file
+  "telephony_group_billing_dashboard_calls": "Appels",
+  "telephony_group_billing_dashboard_suspended_info": "Ce groupe est actuellement suspendu, cliquez <a href=\"{{ billingDepositHref }}\">ici</a> pour augmenter votre dépôt de garantie."
```

Comment:
IMO we should avoid HTML in translation. 
Here it could be splitted into 2 trasnlations : 
1. `Ce groupe est actuellement suspendu.`
2. `Cliquez ici pour augmenter votre dépôt de garantie.` and used in `a` element.